src/ProjectTemplates/Shared/ProjectFactoryFixture.cs

```diff
@@ -44,7 +44,7 @@ public async Task<Project> GetOrCreateProject(string projectKey, ITestOutputHelp
                         DiagnosticsMessageSink = DiagnosticsMessageSink,
                         ProjectGuid = Path.GetRandomFileName().Replace(".", string.Empty)
                     };
-                    project.ProjectName = $"AspNet.{key}.{project.ProjectGuid}";
+                    project.ProjectName = project.ProjectGuid;
```

Comment:
Is this changed because the prefix is no longer needed or?

Reply:
It produces incredibly long paths. With us including rids in the output path, this goes over the windows long path limit:

D:\work\aspnetcore\.dotnet\sdk\5.0.100-preview.6.20310.4\Microsoft.Common.CurrentVersion.targets(4651,5): error MSB3030: Could not copy the file "D:\work\aspnetcore\src\ProjectTemplates\BlazorTemplates.Tests\bin\Debug\net5.0\TestTemplates\AspNet.blazorhostedindividualuld.1zbj1ytj2fl\Client\obj\Release\net5.0\browser-wasm\staticwebassets\AspNet.blazorhostedindividualuld.1zbj1ytj2fl.Client.StaticWebAssets.xml" because it was not found.

^ This file exists. 

I'm not entirely clear on why the name has to be this long. @javiercn has more context, but shortening it seems to work.

Reply:
Not sure what the key was, but can you leave the AspNet prefix? It makes it easy to kill the running projects left behind if you cancel a test run or something similar

Reply:
```suggestion
                    project.ProjectName = $"AspNet.{project.ProjectGuid}";
```